pkg/azurefile/azurefile.go

```diff
@@ -407,10 +410,29 @@ func (d *Driver) Run(endpoint, kubeconfig string, testBool bool) {
 	}
 	d.AddNodeServiceCapabilities(nodeCap)
 
-	s := csicommon.NewNonBlockingGRPCServer()
-	// Driver d act as IdentityServer, ControllerServer and NodeServer
-	s.Start(endpoint, d, d, d, testBool)
-	s.Wait()
+	//setup grpc server
+	opts := []grpc.ServerOption{
```

Comment:
what's the intention for this?
why not wrap `NewNonBlockingGRPCServer` to make logic more clear?

Reply:
the wrapper is not shared among csi projects. 
It is only referred by main once. 
The wrapper doesn't introduce any extra function which grpc server provides.

Reply:
Also, we don't need to add any code for test purposes. the metrics related code can be put to one place. 
and the line of code decreased. 
I think this will make the code clearer.